packages/nc-gui/components/cell/DateTimePicker.vue

```diff
@@ -131,6 +131,10 @@ watch(
   (next) => {
     if (next) {
       onClickOutside(document.querySelector(`.${randomClass}`)! as HTMLDivElement, () => (open.value = false))
+
+      if (!modelValue) {
+        localState.value = dayjs(new Date())
+      }
```

Comment:
as discussed with Raju, the behaviour is a bit off. When I open the datetime picker, the current datetime is expected to be selected. However, we only save the value once we click `OK`, which means the value shouldn't be saved if I un-focus it. Currently the value will be saved anyway.

Reply:
Okay, first I tried to set ` :default-picker-value="dayjs(new Date())"` but this is not woking.
I will check this